src/lib/ui/CommandPalette.svelte

```
@@ -0,0 +1,397 @@
+		};
+
+		addKeyboardListener(handler);
+
+		return () => {
+			removeKeyboardListener(handler);
+		};
+	});
+</script>
+
+{#if $commandPaletteOpen}
+	<!-- svelte-ignore a11y_click_events_have_key_events a11y_no_static_element_interactions -->
```

ESLint

[error] svelte-ignore comment is used, but not warned

(svelte/no-unused-svelte-ignore)